src/__tests__/preact.js

```diff
@@ -0,0 +1,48 @@
+beforeEach(() => {
+  process.env.PREACT = true
+  jest.resetModules()
+  jest.mock('react', () => ({}))
```

Comment:
Why is this necessary? Should it be: `jest.mock('react', () => require('preact'))` instead?

Reply:
Yeah, I've added `preact` as a dev dependency after all. Good idea :clap: